workshops/forms.py

```diff
@@ -346,9 +346,9 @@ class Meta:
         # don't display the 'password', 'user_permissions',
         # 'groups' or 'is_superuser' fields
         # + reorder fields
-        fields = ['username', 'personal', 'middle', 'family', 'may_contact',
-                  'email', 'gender', 'airport', 'affiliation', 'github',
-                  'twitter', 'url', 'notes', 'lessons', 'domains']
+        fields = ['username', 'personal', 'middle', 'family', 'is_active',
+                  'may_contact', 'email', 'gender', 'airport', 'affiliation',
+                  'github', 'twitter', 'url', 'notes', 'lessons', 'domains']
```

Comment:
On Sun, Nov 08, 2015 at 09:04:54AM -0800, Greg Wilson wrote:
> Why not add `is_active` to the end of the list to make a smaller
> diff?

This list also sets the field order in forms, but I don't see an
obvious location for `is_active` in the person form.  One the one
hand, it's fairly important (can the person log in or reset their
password?), which argues for an earlier listing.  On the other hand,
we're unlikely to edit it manually, which argues for a later listing.